packages/cerebral-forms/src/chains/changeField.js

```diff
@@ -0,0 +1,8 @@
+import validateField from '../factories/validateField'
+
+export default [
+  function updateValue ({input, state}) {
```

Comment:
could be replaced with new tagged template based operators, I suppose.
as separate commit